gnomad/gnomadv4_extract_test.py

```diff
@@ -14,7 +14,7 @@
 
 # load list of transcripts
 # a panel of 10 genes selected for initial analysis by the structural-constraint project
-transcripts_file = open('transcripts.json', 'r')
+transcripts_file = open(output_path('transcripts.json'), 'r')
```

Comment:
Python can't directly read from GCS, but we're using `cloudpathlib`'s `AnyPath` in many places.

Reply:
OK that is becoming way too much friction, can we please run this analysis outside of the analysis runner ?

Reply:
This has nothing to do with the analysis-runner. If you submitted this script from your laptop to a Dataproc cluster, you'd run into exactly the same problem.

Also this is a two line change, I wouldn't quite call this "way too much friction"...

```py
from cloudpathlib import AnyPath

with AnyPath(output_path('transcripts.json')).open() as transcripts_file:
    TRANSCRIPTS = json.load(transcripts_file)
```

Reply:
Thanks Leo, that's helpful